docs/01-app/03-api-reference/01-directives/use-cache-remote.mdx

```diff
@@ -0,0 +1,481 @@
+| ------------------------------- | ----------------------------------- | --------------------------------------------------------- | ----------------------------------- |
+| **Works in dynamic context**    | No (requires static context)        | Yes (designed for dynamic contexts)                       | Yes                                 |
+| **Access to `await cookies()`** | No                                  | Yes (but not recommended - use `private` instead)         | Yes                                 |
+| **Access to `await headers()`** | No                                  | Yes (but not recommended - use `private` instead)         | Yes                                 |
+| **After `await connection()`**  | No (won't cache)                    | Yes (will cache)                                          | No                                  |
+| **Stored in cache handler**     | Yes (server-side)                   | Yes (server-side)                                         | No (client-side only)               |
+| **Cache scope**                 | Global (shared)                     | Global (shared)                                           | Per-user (isolated)                 |
+| **Runtime prefetchable**        | N/A (prerendered at build)          | No                                                        | Yes (when configured)               |
+| **Use case**                    | Static, shared content (build-time) | Dynamic, shared content in runtime contexts (per-request) | Personalized, user-specific content |
+
+### When to use each
+
+- **`use cache`**: Use for static content that can be prerendered at build time and doesn't depend on request-specific data.
+- **`'use cache: remote'`**: Use when caching is needed after dynamic data access (`connection()`) or for per-request caching of shared data.
+- **`'use cache: private'`**: Use for user-specific content that needs to be runtime prefetched and depends on cookies, headers, or search params.
+
+## How it works
+
+### Dynamic context detection
+
+When Next.js encounters certain APIs like `connection()`, `cookies()`, or `headers()`, the context becomes "dynamic". In a dynamic context:
+
+1. **Regular `use cache` stops working** - it won't cache anything
+2. **`'use cache: remote'` continues to work** - it caches in the remote cache handler
+3. **Results are stored server-side** in cache handlers like Redis or Vercel Data Cache
+4. **Cached data is shared across requests** - reducing database load and origin requests
+
+> **Good to know:** Without `'use cache: remote'`, functions in dynamic contexts would execute on every request, potentially creating performance bottlenecks. Remote caching eliminates this issue by storing results in server-side cache handlers.
+
+### Storage behavior
+
+Remote caches are **persisted to server-side cache handlers**, such as:
+
+- **Redis** (when configured)
+- **Vercel Data Cache** (on Vercel deployments)
```

Comment:
here, too, listing vercel as a separate entry seems to imply special handling. the clarity for vercel customers here is probably not worth the FUD that special-casing vercel on these pages leads to. in the past @leerob cleaned up the docs from vercel references so it would be nice to avoid them creeping back in. sorry for unsolicited feedback, i hope that’s okay!